src/AppBundle/Controller/BlogController.php

```diff
@@ -34,12 +34,18 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", name="blog_index")
+     * @Route("/{page}", name="blog_index", defaults={"page" = 1}, requirements={"page" : "\d+"})
```

Comment:
This would generate URLs like the following:

```
/blog
/blog/2
/blog/3
...
```

Don't you think those URLs are a bit ugly? I'd prefer to have something like:

```
/blog
/blog/page-2
/blog/page-3
...
```

Or:

```
/blog
/blog/page/2
/blog/page/3
...
```

What do you think?